bleak/backends/scanner.py

```diff
@@ -44,6 +45,9 @@ def __init__(self, **kwargs):
         # Tuple of platform specific data
         self.platform_data: Tuple = kwargs.get("platform_data", ())
 
+        # Tx Power data
+        self.tx_power: Optional[int] = kwargs.get("tx_power",None) # tx_power can't be 0. so 0 may be better default value?
```

Comment:
```suggestion
        self.tx_power: Optional[int] = kwargs.get("tx_power")
```

The value can be from -127 to 126, so 0 is allowed. I think `None` makes sense for "not available".